split.Console/Models/Deck.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+
+namespace split.Console.Models
+{
+    public class Deck
+    {
+        Card[] Cards;
+        int CardsIndex;
+        const int CARDSINDECK = 52;
+
+        public Deck(int numberDecks)
+        {
+            Cards = new Card[numberDecks * CARDSINDECK];
+            CardsIndex = 0;
+
+            for (int i = 0; i < numberDecks; i++)
+            {
+                foreach (SuitType suit in Enum.GetValues(typeof(SuitType)))
+                {
+                    foreach (RankType rank in Enum.GetValues(typeof(RankType)))
+                    {
+                        Cards[CardsIndex++] = new Card(suit, rank);
+                    }
+                }
+            }
+            Shuffle();
+        }
+
+        /// <summary>
+        /// Shuffles Cards and sets index to top of deck
+        /// </summary>
+        public void Shuffle()
+        {
+            Random rand = new Random();
+            for (int n = Cards.Length - 1; n > 0; --n)
+            {
+                int k = rand.Next(n + 1);
+                var temp = Cards[n];
+                Cards[n] = Cards[k];
+                Cards[k] = temp;
+            }
+            CardsIndex = Cards.Length - 1;
+        }
```

Comment:
I will include getter method because i see where you are coming from. Any thoughts on a return value? just boolean? what is a good test criteria for a return value? maybe something like CardIndex + 1 == Cards.Length